def test_kasitli_hata_düzeltilmesi():
    assert True, 'Kasıtlı hata düzeltildi'